components/mobile_vikings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file format isn't used by this repository anymore.

It doesn't seem like a needed or valid PR, so going to close it.

../Frecl